README.md

```diff
@@ -43,7 +43,7 @@ metric = DiagEuclideanMetric(D)
 h = Hamiltonian(metric, ℓπ, ∂ℓπ∂θ)
 int = Leapfrog(find_good_eps(h, θ_init))
 prop = NUTS{MultinomialTS,GeneralisedNoUTurn}(int)
-adaptor = StanHMCAdaptor(n_adapts, Preconditioner(metric), NesterovDualAveraging(0.8, int.ϵ))
+adaptor = StanHMCAdaptor(n_adapts, Preconditioner(metric), NesterovDualAveraging(0.8, int))
```

Comment:
This looks like a breaking change. If so, maybe consider adding a special `NesterovDualAveraging` constructor to maintain backward compatibility, and print a warning message?

Reply:
I think the original construct is still there. It's a conveint wrapper for it.

Reply:
Yes @xukai92's right, this won't break anything.